activities/explaining-codebase-stewardship/codebase-stewardship.md

```diff
@@ -1,5 +1,7 @@
 ---
```

Comment:
This page has a silly URL.

Reply:
I think that for the context it is the right name though. As part of explaining codebase stewardship, there is a level to do that.

Regardless I think fixing this is outside of the scope of this pull request.

Reply:
Yup, that's fine.